Update `readme` for w.org plugins page to match new copy

postphotos opened the pull request:
Updated readme to match new copy that more accurately describes Stream.

derekherman commented on readme.txt:
```diff
@@ -1,24 +1,26 @@
 === Stream ===
-Contributors: lukecarbis, fjarrett, stream, xwp, kasparsd
+Contributors: xwp, kasparsd, derekherman, postphotos
```
This should probably just be `xwp`. Not sure who has access to the stream account on wp.org and this list gives everyone write access to SVN. @kasparsd I can either give you the credentials to the `xwp` account so you can still push new versions of the plugin or keep you on the list but I don't see why I or Leo would be on this list. Thoughts?

postphotos replied:
Agreed. Just pushed a change.

kasparsd replied:
> Not sure who has access to the stream account on wp.org and this list gives everyone write access to SVN

@derekherman This contributors list is only included in the plugin credits and doesn't give SVN access. 

![contributors](https://user-images.githubusercontent.com/169055/86226256-0fa13b00-bb94-11ea-8864-60f7bba8f4e7.png)

This attribution also makes the plugin show up on [user's list of contributions](https://profiles.wordpress.org/kasparsd/#content-plugins) which is nice:

![contributions](https://user-images.githubusercontent.com/169055/86226599-8fc7a080-bb94-11ea-8f3b-73d32efde7fb.png)

The SVN access is managed via Committers under [the plugin settings](https://wordpress.org/plugins/stream/advanced/):

![committers](https://user-images.githubusercontent.com/169055/86226574-86d6cf00-bb94-11ea-982d-d5a10e778036.png)

postphotos commented:
Hey @derekherman - I've added the list of contributors as @kasparsd suggested and removed the other non-repo-committing folks from the list.

I think this is good to merge now.
